pages/index.js

```diff
@@ -78,7 +86,9 @@ export default function HomePage() {
   const [userArmor, setUserArmor] = useState(userData.armor);
```

Comment:
Does it make sense to split up the user object into different states? Why not have one useState for the user? 
` const [user, setUser] = useState(userData)`

Then we could derive the avatar to show without needing to change keep that in state too. 

i.e. 

``` const userAvatarImage = `/assets/avatars/${user.chosenAvater}${user.level}.png` ```